### ISSUE TITLE
Arm support

### ISSUE THREAD

**Issue by jkterry1** [Farama-Foundation/Arcade-Learning-Environment#373]
ARM based Macs are now imminent and ARM in general is becoming increasingly widespread. Are there any plans for ARM support given this?

**Comment by JesseFarebro** [comment]
Yes, this should be as easy as adding a flag to `cibuildwheel`. When `ale-py` is officially released there will be an arm build.

**Comment by JesseFarebro** [comment]
ARM support for macOS wheels is now implemented in #422. There are further issues with Linux aarch64 builds and this will be addressed later.
